near-sdk-sim/src/runtime.rs

```diff
@@ -517,4 +539,13 @@ mod tests {
         runtime.force_account_update("root".into(), &bob_account);
         assert_eq!(runtime.view_account(&"root").unwrap().locked, 10000);
     }
+
+    #[test]
+    fn test_load_states() {
+        let mut genesis = GenesisConfig::default()
+            .add_account("simple-state.repro.testnet".to_string(), 165437999999999999999000)
+            .add_account("multiple-state.repro.testnet".to_string(), 165437999999999999999000)
+            .load_state_records(Path::new("./res/state_records.json"));
```

Comment:
I am not feeling good about dumping testing .json in this repo. There's nothing specifically wrong with it, but it is a minor technical dept. For the test, I suggest the following:

* create a Vec<StateRecord> in memory
* dump it to a temporary file
* use temp file path in the test

Reply:
This json isdumped from testnet, and when user use this API he would also do dump a json from testnet then load it. Sounds more like an integration test here? So i just put in the real dumped json